frontend/.env

```diff
@@ -1,10 +1,10 @@
 # Local Dev Environment
-VITE_APP_NAME=PURIS
```

Comment:
Please revert. These changes MUST NOT be applied to the .env.

The mechanism for docker is the following:
- .env has vite variables
- .env.dockerbuild has the vite variable that maps on an environment variable (VITE_BACKEND_BASE_URL=$BACKEND_BASE_URL)
- src/config.json has the environment variable names and the environment variable to substring in a json format.

When building the container:
- .env.dockerbuild is used
- vite / vue builds the application into a dest folder, that will be served by nginx
- result: for the env. VITE_BACKEND_BASE_URL it won't write a variable value BUT a placeholder into the built files ($BACKEND_BASE_URL)

When building the container, there is a "start-nginx.sh" file that does the following
- collect the environment variables (set for the docker container / set via helm as BACKEND_BASE_URL)
- Looks-up the "to replace string" from config.json (for BACKEND_BASE_URL, it will search for $BACKEND_BASE-URL in the built files)
- does the replacement in the built files
- starts nginx